pyodide-build/pyodide_build/testing.py

```diff
@@ -127,7 +127,9 @@ def wrapped(selenium):  # type: ignore
 
 
 @contextlib.contextmanager
-def set_webdriver_script_timeout(selenium, script_timeout: Optional[Union[int, float]]):
+def set_webdriver_script_timeout(
+    selenium, script_timeout: Optional[Union[int, float, str]]
```

Comment:
Hopefully strings are allowed, because it's in the types elsewhere!

Reply:
I'm guessing that strings are not allowed. Here's the documentation for the selenium function we are calling:
https://selenium-python.readthedocs.io/api.html#selenium.webdriver.remote.webdriver.WebDriver.set_script_timeout
It would be weird if it accepted strings.